temporal-sdk/src/main/java/io/temporal/internal/sync/WorkflowInfoImpl.java

```diff
@@ -80,6 +80,7 @@ public long getRunStartedTimestampMillis() {
   }
 
   @Override
+  @Deprecated
```

Comment:
We don't need to deprecate it, it's a private class because it's located in internal package. If somebody uses it - they will have to deal with that.